boards.txt

```diff
@@ -17,6 +17,7 @@ menu.LoRaWanDebugLevel=LoRaWan Debug Level
 menu.LoopCore=Arduino Runs On
 menu.EventsCore=Events Run On
 menu.MemoryType=Memory Type
+menu.EraseFlash=Erase Flash
```

Comment:
I would recommend changing to `menu.EraseFlash=Erase Flash (on upload)` to avoid any misunderstood.

Reply:
Agree :) done